praxis-isncsci/ui
# [Classification dialog] Responsive dialog

Closes #47 

## Problem

All classification dialog elements need to be represented and responsive.

## Solution

Added responsive elements along with supporting design tokens

## Testing

- [x] Can view a responsive version of the classification dialog

<details>
  <summary>Test case</summary>

  1. Navigate to **Storybook** story `http://localhost:6006/?path=/docs/webcomponents-praxisisncsciclassification--docs`
  2. Confirm that all elements of the classification dialog are displayed
  3. Resize the component
  4. Confirm that the layout is responsive

</details>
